shared/actions/config/index.js

```diff
@@ -409,7 +409,12 @@ const readLastSentXLM = () => {
       logger.info(`Successfully read config stellar.lastSentXLM: ${String(value)}`)
       return createSetLastSentXLM({lastSentXLM: value, writeFile: false})
     })
-    .catch(err => logger.error(`Error reading config stellar.lastSentXLM: ${err.message}`))
+    .catch(
```

Comment:
i think we can just ignore the error. if it doesn't read its ok

Reply:
if it doesn't read and it's set that'd be a bug, though